src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeFeature.cs

```diff
@@ -44,7 +44,9 @@ public static bool IsSupported(string feature)
                 case CovariantReturnsOfClasses:
                 case UnmanagedSignatureCallingConvention:
                 case DefaultImplementationsOfInterfaces:
+#pragma warning disable CA2252
```

Comment:
Don't want to kick off another CI round just for adding a comment here. We can fix it the next time I/we go through this file.